UnitsNet/QuantityFormatter.cs

```diff
@@ -35,16 +36,19 @@ public class QuantityFormatter
         /// <typeparam name="TUnitType">The quantity's unit type, for example <see cref="LengthUnit"/>.</typeparam>
         /// <param name="quantity">The quantity to format.</param>
         /// <param name="format">The format string.</param>
-        /// <param name="formatProvider">The format provider to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
+        /// <param name="formatProvider">The format provider to use for localization and number formatting. Defaults to
+        /// <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
```

Comment:
This needs to be updated to `CultureInfo.CurrentUICulture` after recent PR merge.

Reply:
Done!